Rebus.Idempotency.MySql/MySqlMessageStorage.cs

```diff
@@ -151,6 +152,48 @@ public async Task InsertOrUpdate(MessageData messageData)
             }
         }
 
+        public async Task Verify()
+        {
+            using (var connection = await _connectionHelper.GetConnection())
+            {
+                var tableNames = new List<string>();
+                using (var command = connection.CreateCommand())
+                {
+                    command.CommandText = "select * from information_schema.tables where table_name = @table_name";
```

Comment:
Maybe we could remove this first query here to optimize speed on startup, since the second query will also validate this.
But not blocking at all at this moment!

Reply:
I'll keep it, and if we see really slow startups we can still patch this.